src/ngraph/graph_util.hpp

```diff
@@ -420,7 +419,7 @@ namespace ngraph
 
     // Assert that nodes in the function is colocated and return that placement
```

Comment:
Might as well fix the comment while in the neighborhood.

Reply:
done